src/wrapper/pack.rs

```diff
@@ -22,24 +24,48 @@ pub fn pack_image_cmd(wrapper: &Wrapper, cmdline: Vec<String>)
     let cconfig = try!(wrapper.config.containers.get(&options.name)
         .ok_or(format!("Container {} not found", options.name)));
     let container_ver = wrapper.root.as_ref().unwrap();
-    let root = Path::new("/vagga/base/.roots")
-        .join(container_ver).join("root");
 
-    let mut cmd = Command::new("/vagga/bin/busybox");
-    cmd.stdin(Stdio::null())
+    let tmppath = PathBuf::from(
+        &format!("/vagga/base/.roots/.tmp.{}", &options.name));
```

Comment:
I'm not sure about this name. The name is reserved for building containers. Can we just put it to the tmpfs?